tests/puppeteer/campaigns/upgrade/upgradeShop.js

```diff
@@ -1,6 +1,7 @@
+// Using chai
```

Comment:
Not necessary I guess ?

Reply:
Comment was not in the right place, I just moved it